pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -343,8 +343,10 @@ func addSingleHistogramDataPoint(pt pmetric.HistogramDataPoint, resource pcommon
 	if pt.Flags().HasFlag(pmetric.MetricDataPointFlagNoRecordedValue) {
 		infBucket.Value = math.Float64frombits(value.StaleNaN)
 	} else {
-		cumulativeCount += pt.BucketCounts().At(pt.BucketCounts().Len() - 1)
-		infBucket.Value = float64(cumulativeCount)
+		if pt.BucketCounts().Len() > 0 {
```

Comment:
My brief understanding of this code block is that this value ways always expected, I believe this check should be part of the original if statement as an or, so that there is always a value reported here.

Reply:
I hadn't looked beyond fixing that error, I see now that it should just use the prior if statement to use the already specified default. Thank you for the catch!